modules/ROOT/pages/sap/sap-connector-reference.adoc

```diff
@@ -3,660 +3,716 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+SAP v4.1.3
+
++++
 Class that has an overall configuration for the SAP extension.
++++
+
 
+== Configurations
+---
 [[inbound-config]]
-== Inbound Configuration
+=== Inbound
 
++++
 Default configuration for SAP extension outbound elements.
++++
 
-=== Parameters
+==== Parameters
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-|Name | String | The name for this configuration. Connectors reference the configuration with this name. | |x
-| Connection a| <<inbound-config_simple-connection-provider>>
-| The connection types that can be provided to this configuration. | |x
-| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime only purges instances as it sees fit. |  |
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| <<inbound-config_simple-connection-provider, Simple Connection Provider>>
+ | The connection types to provide to this configuration. | | x
```

Comment:
I think there is an extra space at the beginning of this line. The rest looks good to me!

Reply:
Fixed. Thank you.